src/RestControllers/PatientRestController.php

```diff
@@ -45,19 +62,29 @@ public function put($pid, $data)
         return RestControllerHelper::responseHandler($serviceResult, array("pid" => $pid), 200);
     }
 
-    public function getOne()
```

Comment:
Hello @dixonwhitmire I am late to the party, any reasons for removing the public access modifier? looking to learn

Reply:
hi @prondubuisi , think if you don't specify it defaults to `public` so it was kind of redundant

Reply:
hi @prondubuisi , @stephenwaite , @dixonwhitmire 
this is a good point. psr2 specs rec to explicitly state either public/private/protected.
https://rules.sonarsource.com/php/tag/psr2/RSPEC-1784

Reply:
can't blame travis for this one :)
this could go
https://github.com/openemr/openemr/blob/master/ci/phpcs.xml#L9
or just move to psr 12 :)
https://www.php-fig.org/psr/psr-2/

Reply:
Lol @stephenwaite even Psr-12 requires that visibility MUST be declared https://www.php-fig.org/psr/psr-12/#44-methods-and-functions, that also helps for general consistency. Would have to remove the rule for PHPCS, thanks for pointing that out.

Reply:
hi @prondubuisi, maybe we should we bump our `PSR-2` to `PSR-12`? We will need to checkout the other pieces of `ci/phpcs.xml`.

Reply:
okay @stephenwaite. I am always in for new and shiny things!

Reply:
> this is a good point. psr2 specs rec to explicitly state either public/private/protected.

I think this was just an oversight. I missed it too. I'll have @silvistershebin do a quick fix in his PR for procedures. Thanks for catching but my goodness, you'd think the world was ending:)

Reply:
The old man with a walking stick always cracks me up  with his Sarcasm!

Reply:
lol, yep and it has gotten me in trouble in the past. So, wouldn't rec for everyone.
I do love the smiley face though, it begs the audience to question umm "Was he joking" or did he really mean it.:) See.

Reply:
The world is ending!

![tenor (10)](https://user-images.githubusercontent.com/278968/79719920-6c2ed280-8294-11ea-9b73-849bb3cced2e.gif)

After the world ends, be good to place public back on these functions. This is something I usually force for classes that get placed in src/ directory (ie. PSR4 classes), but I was distracted with the end of the world coming and all.

Reply:
Thanks for the discussion and lively feedback! 

On my next round of changes I will ensure that we have the appropriate access modifier on the components being modified. I will also go ahead and add `public` back in if it's not done by the time I submit a PR.